packages/mui-lab/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/lab",
-  "version": "7.0.0-beta.7",
+  "version": "7.0.0-rc.0",
```

My bad, it should be 

```suggestion
  "version": "7.0.0-beta.8",
```